ckanext/datarequests/auth.py

```diff
@@ -112,7 +112,8 @@ def delete_datarequest(context, data_dict):
 
 
 def close_datarequest(context, data_dict):
-    return auth_if_creator(context, data_dict, constants.SHOW_DATAREQUEST)
+    # Close data request feature is removed in this project.
```

Comment:
FYI CKAN does not check auth functions for sysadmin users and automatically returns true because they have access to everything.
Sysadmin users will be able to use the API to close a data request.
To prevent sysadmin users from being automatically authorised add the decorate `auth_sysadmins_check`